pkg/runtime/executor.go

```diff
@@ -139,3 +168,68 @@ func (e *Executor) Execute() (results Output, err error) {
 	// successful
 	return results, nil
 }
+
+// read the config file and update scan and skip rules
+func (e *Executor) initScanAndSkipRules() error {
+	if e.configFile != "" {
+		configData, err := config.LoadConfig(e.configFile)
+		if err != nil {
+			zap.S().Error("error loading config file", zap.Error(err))
+			return err
+		}
+
+		if configData.Has(rulesKey) {
+
+			data := (configData.Get(rulesKey)).(*toml.Tree)
+
+			// read scan rules in the toml tree
+			if err := initRules(e, data, scanRulesKey); err != nil {
+				return err
+			}
+
+			// read skip rules in the toml tree
+			if err := initRules(e, data, skipRulesKey); err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+func initRules(e *Executor, tree *toml.Tree, key string) error {
+	rules, err := getRulesInTomlTree(tree, e.configFile, key)
+	if err != nil {
+		zap.S().Error("error reading config file", zap.Error(err))
+		return err
+	}
+	if len(rules) > 0 {
+		if key == scanRulesKey {
+			e.scanRules = append(e.scanRules, rules...)
+		} else {
+			e.skipRules = append(e.skipRules, rules...)
+		}
+	} else {
+		zap.S().Debugf("key '%s' not found in the config file: %s", key, e.configFile)
+	}
+	return nil
+}
+
+func getRulesInTomlTree(tree *toml.Tree, configFile, key string) ([]string, error) {
```

Comment:
Another good candidate for `rules.go`.

But, I feel this should function should be a part of the `config` package. This function can be a helper for returning rules from the `config` package.

Reply:
this was removed as part of config reader refactor